GoedeDoelenHelpen/ClientApp/src/app/home/home.module.ts

```diff
@@ -0,0 +1,42 @@
+// Modules
```

Comment:
If you want a architecture with multiple modules you should also a module for shared services, modules and comonents